### PR TITLE
Update the metrics autoconfigure code with support for duration units.

### PR DESCRIPTION
**Pull request by jkwatson**
Even though metrics aren't stable yet, it seems good to have some configuration consistency.

### CODE REVIEW

File: sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java [hunk 1]
```diff
@@ -92,9 +92,9 @@ private static void configureIntervalMetricReader(
         IntervalMetricReader.builder()
             .setMetricProducers(Collections.singleton(meterProvider))
             .setMetricExporter(exporter);
-    Long exportIntervalMillis = config.getLong("otel.imr.export.interval");
+    Duration exportIntervalMillis = config.getDuration("otel.imr.export.interval");
```
Comment (breedx-splk) [inline sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java:95]: probably rename that variable here too?

**Review by breedx-splk — CHANGES_REQUESTED** [review]
Cool. I think that variable wants a little rename.

**Review by anuraaga — APPROVED** [review]
Just @breedx-splk's nit but LGTM

File: sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java (hunk 1 above)
Comment (jkwatson) [reply sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java]: done

**Pull request merged**